docs/src/inputs/button/button.jsx

```diff
@@ -188,6 +189,21 @@ function ElementsButton() {
                 <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
                     {require('!!raw-loader!./buttonWidthExample').default}
                 </Highlighter>
+
+                {/* Pill */}
+                <Header anchor="width" size="large" style={{ marginTop: '55px' }} sub>
+                    Pill
+                    <Header.Subheader>
+                        A button can pill form.
+                    </Header.Subheader>
+                </Header>
+
+                <ButtonPillExample />
+
+                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
+                    {require('!!raw-loader!./buttonPillExample').default}
+                </Highlighter>
+
```

Comment:
Please use the new documentation components. We can get on a call to over this. Ping me when you can.